tpm2/tpm2.go

```diff
@@ -1456,6 +1456,33 @@ func NVReadLock(rw io.ReadWriter, owner, handle tpmutil.Handle, authString strin
 	return err
 }
 
+// PolicyNV sets NV object state binding for authorization on a session.
+// auth and password must represent a handle and authorization value which is
+// allowed to read the NV index. For example, if the NV index has the
+// "OwnerRead" and "AuthRead" attributes, you can pass either RH_OWNER and the
+// Owner password, or the NV index and the NV's auth value.
+// This function does not support binding to NV indices which cannot be read
+// with an auth value.
+func PolicyNV(rw io.ReadWriter, auth, index, session tpmutil.Handle, password string, operandB tpmutil.U16Bytes, offset uint16, operation EO) error {
```

Comment:
Question about parameter ordering, we currently aren't super consistent with how we handle parameter ordering for commands that Read/Write NVData. Per the spec, you always need two handles (`authHandle` and `nvIndex`) and an authorization area corresponding to `authHandle`. Here's the variants we currently have:

- 1 param: `nvIndex`
  - assumes `authHandle = nvIndex`, assumes empty password authorization
  - [`NVRead`](https://pkg.go.dev/github.com/google/go-tpm@v0.3.2/tpm2#NVRead)
- 2 params: `nvIndex`, `authString`
  - assumes `authHandle = nvIndex`, assumes password authorization
  - [`NVIncrement`](https://pkg.go.dev/github.com/google/go-tpm@v0.3.2/tpm2#NVIncrement)
- 3 params: `nvIndex`, `authHandle`, `authString`
  - assumes password authorization
  - [`NVReadEx`](https://pkg.go.dev/github.com/google/go-tpm@v0.3.2/tpm2#NVReadEx)
- 3 params: `authHandle`, `nvIndex`, `authString`
  - assumes password authorization
  - [`NVWrite`](https://pkg.go.dev/github.com/google/go-tpm@v0.3.2/tpm2#NVWrite)
  - [`NVReadLock`](https://pkg.go.dev/github.com/google/go-tpm@v0.3.2/tpm2#NVReadLock)
  - [`NVWriteLock`](https://pkg.go.dev/github.com/google/go-tpm@v0.3.2/tpm2#NVWriteLock)
- 3 params: `authString`, `authHandle`, `nvIndex`
  - assumes password authorization
  - [`NVUndefineSpace`](https://pkg.go.dev/github.com/google/go-tpm@v0.3.2/tpm2#NVUndefineSpace)
- 3 params: `authHandle`, `nvIndex`, `authArea`
  - assumes nothing!
  - [`NVUndefineSpaceEx`](https://pkg.go.dev/github.com/google/go-tpm@v0.3.2/tpm2#NVUndefineSpaceEx)
  - [`NVWriteEx`](https://pkg.go.dev/github.com/google/go-tpm@v0.3.2/tpm2#NVWriteEx)

Obviously, this is something we should clean up for the next breaking change.

So I think the best signature would be something like:
```go
func PolicyNV(rw io.ReadWriter, authHandle, nvIndex tpmutil.Handle, authArea AuthCommand, session tpmutil.Handle, ...) error
```

Reply:
Could we just use `[]byte` for `operandB`? And only use the `tpmutil.U16Bytes` internally?

Reply:
Could we make the last three parameters here a helper struct? They only ever appear together.